trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacility.js

```diff
@@ -1,67 +0,0 @@
-const { formattedNumber } = require('../../../../utils/number');
```

Comment:
This file is renamed and edited to `mapGefFacilitySnapshot` (hence the large diff showing up) - same with the test file